includes/WikiDiscoverWikisPager.php

```diff
@@ -76,7 +76,7 @@ function formatValue( $name, $value ) {
 				break;
 			case 'wiki_category':
 				$wikicategories = array_flip( $this->getConfig()->get( 'CreateWikiCategories' ) );
-				$formatted = $wikicategories[$row->wiki_category];
+				$formatted = $wikicategories[$row->wiki_category] ?? '';
```

Comment:
If a category is unknown, wouldn't you say it is uncategorised rather than 'blank'?